Ginger/GingerCoreCommon/GeneralLib/JSONLib/JsonSchemaTools.cs

```diff
@@ -234,21 +250,39 @@ private static object GenerateJsonObjectFromJsonSchema4(JsonProperty value, List
                     break;
 
                 case JsonObjectType.String:
-                    output = new JValue("<name>");
+                    if (value.Example == null && value.IsEnumeration && value.Enumeration?.Count  >0)
+                    {
+                        output = new JValue(value.Enumeration.FirstOrDefault());
+                        break;
+                    }
+                    output = new JValue(value.Example ?? $"<{value.Type}>");
                     break;
                 case JsonObjectType.Number:
                     output = new JValue(1);
                     break;
                 case JsonObjectType.Integer:
-                    output = new JValue(1);
+                    output = new JValue(value.Example ?? 1);
```

Comment:
please set example value for Number and boolean if present just like it is done for Integer

Reply:
Added.